### DISCUSSION TITLE
Import - image that I don't know what it means (Step-Over)

### DISCUSSION THREAD

**Discussion by shackrock** [teslamate-org/teslamate#1249] [Q&A] [answered]
On import, I have mostly green check marks, but some "step over" images.  Does this mean they were skipped?

![image](https://user-images.githubusercontent.com/559901/105709753-261e3380-5ee4-11eb-8631-27849252e9e0.png)

**Comment by adriankumpf** [comment, marked as answer]
Yes, that's correct.

**Reply by shackrock** [reply]
Got it, and I assume skipped "for good reason" ?  (I saw on the import docs that if teslamate already had data, it used that instead of the imported data).
